zio-http-gen/src/test/resources/inline_schema_any_and_any_object.yaml

```diff
@@ -0,0 +1,49 @@
+info:
+  title: Animals Service
+  version: 0.0.1
+tags:
+  - name: Animals_API
+paths:
+  /api/v1/zoo/{animal}:
+    get:
+      operationId: get_animal
+      parameters:
+        - in: path
+          name: animal
+          schema:
+            type: string
+          required: true
+      tags:
+        - Animals_API
+      description: Get animals by species name
+      responses:
+        "200":
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Animal'
+          description: OK
+        "500":
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/HttpError'
+          description: Internal Server Error
+openapi: 3.0.3
+components:
+  schemas:
+    Animal:
+      type: object
+      required:
+        - name
+        - eats
+        - extra_attributes
+      properties:
+        name:
+          type: string
+        eats: {}
+        extra_attributes:
+          type: object
+          additionalProperties: true
```

Comment:
should also test/implement for 
`additionalProperties: {}` and without additional properties at all?

Reply:
Sorry, help me out. This dynamic fields open api always confuses me

Reply:
this is straight out of the [docs](https://swagger.io/docs/specification/v3_0/data-models/data-types/):
<img width="755" alt="Screenshot 2024-12-02 at 17 05 06" src="https://github.com/user-attachments/assets/ab8c7bfc-3e41-4b66-8d5b-a7c820868155">